data.sql

```diff
@@ -166,3 +166,91 @@ UPDATE animals SET owner_id = 3 WHERE name IN  ('Devimon' ,'Plantmon');
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander' ,'Squirtle' ,'Blossom');
 UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon' ,'Boarmon');
 COMMIT;
+
+
+BEGIN;
+INSERT INTO vets (name, age, date_of_graduation) 
+  VALUES ('William Tatcher',  45, '2000-04-23'),
+    ('Maisy Smith', 26, '2019-01-17'),
+    ('Stephanie Mendez', 64, '1981-05-04'),
+    ('Jack Harkness', 38, '2008-06-08');
+END;
+
+
+-- Insert data into the 'specializations' table
+
+BEGIN;
+INSERT INTO specializations (vet_id, species_id) 
+  VALUES ((SELECT id FROM vets WHERE name='William Tatcher'),
+     (SELECT id FROM species WHERE name='Pokemon')),
+    ((SELECT id FROM vets WHERE name='Stephanie Mendez'),
+     (SELECT id FROM species WHERE name='Digimon')),
+    ((SELECT id FROM vets WHERE name='Stephanie Mendez'),
+     (SELECT id FROM species WHERE name='Pokemon')),
+    ((SELECT id FROM vets WHERE name='Jack Harkness'),
+     (SELECT id FROM species WHERE name='Digimon'));    
+END;
```

Comment:
- [ ] Let us use this way again to insert the data into specializations

INSERT INTO specializations (species_id, vet_id)

 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));